.github/workflows/main.yml

```diff
@@ -98,10 +98,19 @@ jobs:
           distribution: "temurin" # See 'Supported distributions' for available options
           java-version: "19"
       - name: Apalache integration tests
-        run: cd ./quint && npm run apalache-integration
+        run: |
+          cd ./quint
+          # Fetch the latest apalache release
+          make -C .. apalache
+          # Run the "disintegration" tests
```

Comment:
```suggestion
          # Run the "integration" tests
```

Reply:
What are "disintegration" tests? :)

Reply:
They test ways that the integration fails :)